src/tidy-html5.mk

```diff
@@ -0,0 +1,31 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := tidy-html5
+$(PKG)_WEBSITE  := http://www.html-tidy.org/
+$(PKG)_DESCR    := HTML/XML syntax checker and reformatter
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 5.4.0
+$(PKG)_CHECKSUM := a2d754b7349982e33f12d798780316c047a3b264240dc6bbd4641542e57a0b7a
+$(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
+$(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := https://github.com/htacg/tidy-html5/archive/$($(PKG)_VERSION).tar.gz
```

Comment:
After #1712, you can replace $(PKG)_FILE, $(PKG)_SUBDIR, $(PKG)_URL, and $(PKG)_UPDATE with:

$(PKG)_GH_CONF